### PR TITLE
Feature#79 create access token for rest api

### PR DESCRIPTION
**Pull request by OlehHnyp**
Implemented JWT token authentication for REST API and created client for it.

### CODE REVIEW

File: ProjectShop/OrderItems/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    operations = [
+        migrations.CreateModel(
+            name='OrderItems',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('quantity', models.PositiveIntegerField(verbose_name='quantity')),
+                ('order_id', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='order.order')),
+            ],
+        ),
+    ]
```
Comment (v-lavrushko) [inline ProjectShop/OrderItems/migrations/0001_initial.py:24]: I don`t know what we are supposed to do with migration files

**Review by v-lavrushko — COMMENTED** [review]
1) Have you rebased your branch over some other branch?
2) I am thinking about moving all telegram modules out of Django directory

File: ProjectShop/UserApp/models.py [hunk 2]
```diff
@@ -74,8 +75,10 @@ def has_perm(self, perm, obj=None):
...
 
     def has_module_perms(self, app_label):
+        if app_label == 'token_blacklist':
+            return self.is_superuser
         return self.role
 
     @property
     def is_staff(self) -> int:
-        return self.role
+        return not self.is_bot and self.role
```
Comment (strangerok1507) [inline ProjectShop/UserApp/models.py:84]: советую not оборачивать в скобки, будет удобнее
```suggestion
        return (not self.is_bot) and self.role
```

File: ProjectShop/OrderItems/migrations/0001_initial.py (hunk 1 above)
Comment (strangerok1507) [reply ProjectShop/OrderItems/migrations/0001_initial.py:24]: Что ты имеешь ввиду?

**Pull request merged**